metabase/metabase
# CI for Embedding SDK

Child of https://github.com/metabase/metabase/issues/40241

### Description

This adds Github actions workflow to build and publish Embedding SDK package.
Job will be triggered if `embedding-sdk-build` label has beed assigned to the PR.

### How to verify

Check previous pipelines - https://github.com/metabase/metabase/actions/workflows/embedding-sdk.yml
Check built package page - https://github.com/metabase/metabase/pkgs/npm/embedding-sdk-react

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
